parser/src/lexer.rs

```diff
@@ -456,6 +458,27 @@ where
         }
     }
 
+    fn uncicode_literal(&mut self, literal_number: usize) -> Result<char, LexicalError> {
+        let location = self.get_pos();
+        let mut it: Vec<u32> = Vec::new();
```

Comment:
I suggest to pick a better name for the variable `it`. Perhaps `digits` would be better?

Reply:
Thanks for the suggestion,  I remove the `it` and just put digit into `p`.